solutions/src/main/java/ru/sberbank/school/task07/FileParserImpl.java

```diff
@@ -0,0 +1,24 @@
+package ru.sberbank.school.task07;
+
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FileParserImpl implements FileParser {
+    @Override
+    public List<String> parse(String pathToFile) {
+        List<String> list = null;
+        try {
+           list = Files.lines(Paths.get(pathToFile), StandardCharsets.UTF_8)
```

Comment:
&#39;try&#39; child has incorrect indentation level 11, expected level should be 12.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-494107714>*